index.html

```diff
@@ -320,9 +320,59 @@ <h2>Skills</h2>
                     </article>
                 </article>
             </article>
+            <!-- contact section -->
+            <section class="main-container bottom" >
+                <div class="contact-upper-bar">
+                    <p class="contact-heading">
+                        I'm always interested in hearing 
+                        about new projects, so if you'd 
+                        like to chat please get in touch.
+                    </p>
+                </div>
+                <div class="contact-bottom-bar">
+                    <form action="https://formspree.io/f/mlekqgrr" method="POST">
+                        <div class="input-box">
+                            <input type="text" class="input" name="full-name" placeholder="Full name" required>
+                            
+                        </div>
+                        <div class="input-box">
+                            <input type="email" class="input" name="email" placeholder="Email adress" required>
+                            
+                        </div>
+                        <div class="input-area">
```

Comment:
- [ ] As part of the requirement for this milestone is to try and replicate the given design ad much as you can, I think your text inputs need some styling adjustments, 
- [ ] Your text inputs seem not to be full width and it seems that you added a line below then, Looking at the Figma design, the best way to get the right design, is to give the inputs full widths and the line below them can be achieved by adding a border-bottom to the input itself, I'll suggest you update you UI with this suggestion :+1: :+1: 

- -Screenshot:
![Screenshot from 2023-06-14 09-50-57](https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/107632780/2abe71c3-e61f-48e8-8dbf-eb8513a60b1d)